src/tag/src/components/Tag.js

```diff
@@ -5,29 +5,32 @@ import * as Colors from '../../../colors';
 import { getStyleVariation } from '../../../utils/StyleUtils';
 
 const {
+  BLUE,
   GREEN,
+  MAGENTA,
   NEUTRAL,
   PURPLE,
-  RED,
-  YELLOW,
+  ORANGE,
 } = Colors;
 
 const getBackgroundColor = getStyleVariation('mode', {
-  danger: RED.R300,
+  danger: MAGENTA.M00,
   neutral: NEUTRAL.N700,
+  info: BLUE.B00,
   primary: PURPLE.P300,
-  secondary: PURPLE.P100,
-  success: GREEN.G300,
-  warning: YELLOW.Y300
+  secondary: PURPLE.P00,
+  success: GREEN.G00,
+  warning: ORANGE.O00
 }, NEUTRAL.N100);
 
 const getFontColor = getStyleVariation('mode', {
-  danger: 'white',
-  neutral: 'white',
-  primary: 'white',
-  secondary: PURPLE.P300,
-  success: 'white',
-  warning: NEUTRAL.N700
+  danger: MAGENTA.M400,
+  neutral: NEUTRAL.N100,
```

Comment:
Can we just use white for the neutral font?

<img width="227" alt="Screen Shot 2020-10-09 at 8 22 37 AM" src="https://user-images.githubusercontent.com/440299/95601666-a178d980-0a08-11eb-8a1b-ec7ae1e88287.png">